hello/hello_activity_choice.py

```diff
@@ -0,0 +1,121 @@
+import asyncio
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import List
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+# Activities that will be called by the workflow
+
+
+@activity.defn
+async def order_apples(amount: int) -> str:
+    return f"Ordered {amount} Apples..."
+
+
+@activity.defn
+async def order_bananas(amount: int) -> str:
+    return f"Ordered {amount} Bananas..."
+
+
+@activity.defn
+async def order_cherries(amount: int) -> str:
+    return f"Ordered {amount} Cherries..."
+
+
+@activity.defn
+async def order_oranges(amount: int) -> str:
+    return f"Ordered {amount} Oranges..."
+
+
+@dataclass
+class ShoppingListItem:
+    fruit: str
```

Comment:
Why not enum?

Reply:
I don't think it adds anything when everything is right next to each other. Might as well use string consts. But I can add enums I guess.

Reply:
Added enums. Also opened https://github.com/temporalio/sdk-python/pull/74.

Reply:
👍